packages/upload/test/adding-files.test.js

```diff
@@ -35,11 +35,11 @@ describe('file list', () => {
 
   describe('with add button', () => {
     let input;
-    let addFiles;
+    let addButton;
     let inputClickSpy;
 
     beforeEach(() => {
-      addFiles = upload.$.addFiles;
+      addButton = upload.querySelector('[slot="add-button"]');
```

Comment:
Could be `upload._addButton` if applying the comment above. I think we used both approaches in recent PRs, not sure which one is deemed preferrable.

Reply:
Done.